Xamarin.Forms.Platform.MacOS/Cells/CellRenderer.cs

```diff
@@ -45,7 +46,15 @@ protected void WireUpForceUpdateSizeRequested(ICellController cell, NSView nativ
 
 			_onForceUpdateSizeRequested = (sender, e) =>
 			{
-				//TODO: Implement ForceUpdateSize
+                var index = tableView?.RowForView(nativeCell);
```

Comment:
we will need to fix the styling, we use tabs.

Reply:
Fixed!